src/cluecode/copyrights.py

```diff
@@ -1969,6 +1969,9 @@ def as_str(cls, node, ignores=frozenset(), include_allrights=False):
     # Copyright (c) 2017 Contributors et.al.
     COPYRIGHT: { <COPY> <COPY> <YR-RANGE> <CONTRIBUTORS> <OTH> } #2276
 
+    #Copyright (c) 2020 Contributors as noted in the AUTHORS file
+    COPYRIGHT: { <COPY> <COPY> <YR-RANGE> <CONTRIBUTORS> <NN>* <IN>? <NN>* <CAPS|AUTHS|ATH> <JUNK> }
+
```

Comment:
Just to be sure the trailing `<JUNK>` is for `file` ... may be we can tag `file` as its own? I am concerned that JUNK may catch too much ... on the other hand you have a pretty specific rule.